src/IMG_gif.c

```diff
@@ -30,9 +30,11 @@
 #endif
 
 // By default, non-indexed surfaces will be converted to indexed pixels using octree quantization.
-#if defined(SAVE_GIF)
-#define SAVE_GIF_OCTREE 1
-#endif
+#if SAVE_GIF
+    #ifndef SAVE_GIF_OCTREE
+        #define SAVE_GIF_OCTREE 1
+    #endif /* SAVE_GIF_OCTREE */
+#endif /* SAVE_GIF */
```

Comment:
`#endif` comments serve nothing but hurting readers' eyes when they are this much close to their parent `#if`s